framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -406,6 +408,10 @@ private void addInitialFieldValues(S store, ClassTree classTree, MethodTree meth
                 // If it's not a constructor, use the declared type if the receiver of the method is
                 // fully initialized.
                 boolean isInitializedReceiver = !isNotFullyInitializedReceiver(methodTree);
+                boolean isStaticField = ElementUtils.isStatic(varEle);
```

Comment:
Could this be moved after the initialization of `varEle`?
I assume a constructor does not count as a static method, but please verify. If it is, you can add `&& !isConstructor` to the initializer of `isStaticMethod`.

Reply:
I checked, constructor is not regarded as static method here.

Reply:
So can you move the boolean and check to earlier in the code?

Reply:
So can you move the boolean and check to earlier in the code?

Reply:
changed it